### ISSUE TITLE
Split library into subspecs for Cocoapods

### ISSUE THREAD

**Issue by jondwillis** [RxSwiftCommunity/RxSwiftExt#197]
### Name and description
I suggest splitting this library into subspecs.

### Motivation for inclusion
Splitting into subspecs may reduce overall framework size for those that only want one or a handful of functions, at a minor cost of code simplicity.

### Example of use

pod 'RxSwiftExt/Core'
pod 'RxSwiftExt/PausableBuffered'

**Comment by freak4pc** [comment]
We had discussions around this, but it doesn't really make sense at the granularity you're suggesting. 

Having subspecs per-operator isn't something feasible or sensible in my honest opinion.
I don't think that cost is _minor_ at all, definitely not in the maintainability side of things.

What were your advanced thoughts here?

**Comment by MortyMerr** [comment]
I don't think that one operator per subspec is a good idea either. Might be quite confusing for user :)
